### DISCUSSION TITLE
error during build: EACCES: permission denied, copyFile

### DISCUSSION THREAD

**Discussion by yunics-highfield** [sapphi-red/vite-plugin-static-copy#87] [Q&A] [answered]
When I used `vite-plugin-static-copy` for building, I got an error below. Why does this happen? And how do I solve it?

```
error during build:
EACCES: permission denied, copyFile '/home/projects/vitejs-vite-mmjbfj/src/version.json' -> '/version.json'
```

**Reproducer**
https://stackblitz.com/edit/vitejs-vite-mmjbfj?file=vite.config.js,src%2Fversion.json&terminal=dev

**Reproduction steps**
1. Stop `vite`.
2. Run `vite run build`.

**Comment by sapphi-red** [comment]
It didn't reproduce on my Windows machine or stackblitz.

**Reply by yunics-highfield** [reply, marked as answer]
Sorry about that. I modified the stackblitz by accident (I modified it back.). And I solved it myself. If you set root path to `dest`, it must be `""`, not `"/"`. If you set `"/"` it might fail to build in some files.
